src/current/_includes/v22.2/sidebar-data/manage.json

```diff
@@ -220,29 +220,29 @@
               ]
             },
             {
-              "title": "Managing AWS PrivateLink for a Dedicated Cluster",
+              "title": "AWS PrivateLink for Dedicated Clusters",
```

Comment:
In the preview, this title hadn't updated.... The change is here, so maybe this is just some strange preview thing,